RELEASE.md

```diff
@@ -1,3 +1,12 @@
+# Release 1.12.2
+
+The 1.12.2 minor series tracks TensorFlow 1.12.
```

Comment:
Maybe let's just remove this line, and also remove it for 1.12.1, since as discussed yesterday it doesn't really make sense for patch releases?